core/src/main/java/com/nisovin/magicspells/spells/targeted/SpawnEntitySpell.java

```diff
@@ -488,6 +491,7 @@ private class AttackMonitor implements Listener {
 		private final LivingEntity monster;
 		private final String[] args;
 		private final float power;
+		private final Boolean attackSpellCancelsDefaultAttack;
```

Comment:
Should be `boolean`, not `Boolean`. And as mentioned prior, odd that this option supports replacement.